frontend/src/components/ListDetail.js

```diff
@@ -62,13 +72,14 @@ export const ListDetail = ({ id, name, completed, items, handleUpdate }) => {
             <input className="fs-3" rows="2" value={currentName} onChange={(e) => setCurrentName(e.target.value)} onKeyPress={(e) => handleListUpdate(e)}></input>
             <ul className="list-group">
                 {itemsState.map((item, index) => {
-                    return <Item id={item.id} name={item.name} completed={item.completed} setCompleted={handleSetCompleted} key={index} />
+                    return <Item id={item.id} name={item.name} completed={item.completed} listId={id} setCompleted={handleSetCompleted} key={index} deleteItem={handleDeleteItem} />
```

Comment:
Do we need to pass the listId into the Item?

Reply:
addressed